### PR TITLE
Cluster deployment edit

### PR DESCRIPTION
**Pull request by matkuliak**
- edit the guide to include re-integrated AWS and Azure subscriptions payment option

## Summary of the changes / Why this is an improvement

https://github.com/crate/cloud/issues/748

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Comment by matkuliak** [comment]
@kojinkai please, do you feel that the
```
AWS/Azure
---------
```
section is sufficient? I haven't worked with those subscriptions yet, but I imagine that it should be just a simple confirmation once brought to those marketplaces, correct? Should I elaborate on that more?

File: docs/cluster-deployment/stripe.rst [hunk 1]
```diff
@@ -5,14 +5,15 @@
...
-In this tutorial, we will provide a step by step guide to deploying a cluster
+In this tutorial, we will provide a step-by-step guide to deploying a cluster
 from scratch. Our recommended procedure assumes you have completed the
 :ref:`signup process <sign-up>` and have created an organization (either
 through AzureAD setup or through the Amazon Cognito signup process). The
 simplest way to then deploy a cluster quickly is to subscribe to CrateDB Cloud
 using your credit card. Here we will explain how to do so, from the initial
 subscribing to the deployment itself. Our payment processing and billing are
-powered by `Stripe`_.
+powered by `Stripe`_. It is also to use your AWS or Azure subscription as a
```
Comment (kojinkai) [inline docs/cluster-deployment/stripe.rst]: did you mean 
```suggestion
powered by `Stripe`_. It is also possible to use your AWS or Azure subscription as a
```

File: docs/cluster-deployment/stripe.rst [hunk 2]
```diff
@@ -43,7 +44,7 @@ First, select a `region`_. Your cluster will be deployed in this region.
    :alt: Region selection
 
 .. NOTE::
-    Your choice of region may affect the price of the cluster. Refer to the
+    Your choice of the region may affect the price of the cluster. Refer to the
```
Comment (kojinkai) [inline docs/cluster-deployment/stripe.rst]: I don't think this change is required. 

```
your choice of region
```
is correct from my understanding

File: docs/cluster-deployment/stripe.rst [hunk 3]
```diff
@@ -61,26 +62,26 @@ accordingly in the middle panel that shows the node specification.
...
-the current CPU and RAM values of your cluster, which is simply the compute
+the current CPU and RAM values of your cluster, which are simply the compute
 values of the plan you have chosen multiplied by the number of nodes.
 
 CrateDB Cloud separates storage configuration from compute configuration. You
 can separately configure the desired storage for your cluster by using the
 dropdown menu under 'Storage' in the 'Node specification' panel. The range of
 eligible storage sizes varies by subscription plan. The storage you have chosen
-for your cluster is also reflected in the cluster overview in the right-hand
+for your cluster is also reflected in the cluster overview on the right-hand
```
Comment (kojinkai) [inline docs/cluster-deployment/stripe.rst]: I don't think this change is required
```
in the right-hand panel
```

reads correctly to me

File: docs/cluster-deployment/stripe.rst [hunk 4]
```diff
@@ -115,7 +116,7 @@ hour and per month in the blue bar at the bottom of the screen.
 Set up your account
 -------------------
 
-At the bottom of the deployment screen you can configure your account settings.
+At the bottom of the deployment screen, you can configure your account settings.
```
Comment (kojinkai) [inline docs/cluster-deployment/stripe.rst]: I don't think the comma is required here

File: docs/cluster-deployment/stripe.rst [hunk 5]
```diff
@@ -135,19 +136,42 @@ Provide billing information
...
+deploy your cluster. Do not forget to accept financial authorization by ticking
+the box at the bottom.
 
 .. image:: ../_assets/img/stripe-billing.png
    :alt: Billing information screen
 
+AWS/Azure
+---------
+
+The other payment option is to use AWS or Azure marketplace subscription.
```
Comment (kojinkai) [inline docs/cluster-deployment/stripe.rst]:
```suggestion
The other payment option is to use an AWS or Azure marketplace subscription.
```

File: docs/cluster-deployment/stripe.rst [hunk 6]
```diff
@@ -135,19 +136,42 @@ Provide billing information
...
+Simply choose the one you'd like to use and you will be brought to
```
Comment (kojinkai) [inline docs/cluster-deployment/stripe.rst]: If you wanted to be more specific in your description perhaps you could say either
```suggestion
Simply choose the one you'd like to use and you will be taken to
```
or perhaps better still

```suggestion
Simply choose the one you'd like to use and you will be forwarded to
```

File: docs/cluster-deployment/stripe.rst [hunk 7]
```diff
@@ -135,19 +136,42 @@ Provide billing information
...
+   :alt: Billing information screen
+
+---
+
 The payment and billing information you have submitted will be saved in the
 Billing tab of the Organization overview screen in the CrateDB Cloud Console
 (i.e., the fifth tab from the left on the same screen you arrived at).
 
-You can at any time check your current accumulated bill at the bottom left of
+You can, at any time, check your current accumulated bill at the bottom left of
```
Comment (kojinkai) [inline docs/cluster-deployment/stripe.rst]: To resolve any grammatical ambiguity here you could say something like

```
Your current accumulated bill is shown in the billing meter in the bottom left of the CrateDB Cloud Console screen:
```

or also if you prefer the original sentence you could say something like

```
You can check your current accumulated bill at any time by viewing the billing meter in the bottom left of the CrateDB Cloud Console screen
```

**Review by kojinkai — APPROVED** [review]
Looks good to me 👍🏼

**Comment by kojinkai** [comment]
> @kojinkai please, do you feel that the
> 
> ```
> AWS/Azure
> ---------
> ```
> 
> section is sufficient? I haven't worked with those subscriptions yet, but I imagine that it should be just a simple confirmation once brought to those marketplaces, correct? Should I elaborate on that more?

@matkuliak Well, my opinion is that I don't think you need to elaborate more on the marketplace journey, it's fine as it is I think.

**Comment by matkuliak** [comment]
Thanks for the review ✨

**Pull request merged**